D4RL (Mujoco).rst

```diff
@@ -0,0 +1,343 @@
+
+-  The vector consists of physical information(3D position, orientation, and joint angles etc.), and the specific size is \ ``(N, )``\, which is determined by the environment. The data type is \ ``float64``
+-  `Fujimoto <https://github.com/opendilab/DI-engine/blob/main/dizoo/d4rl/entry/d4rl_cql_main.py>`__ mentions that doing obs norm for the d4rl dataset will improve the stability of offline training.
+
+
+Action Space
+----------------
+
+-  The vector consists of physical information(torque etc.), which is often the continuous action space.（N varies with specific subenvironments), the data type is \ ``float32``\ ，需要传入 np 数组（例如动作为\ ``array([-0.9266078 , -0.4958926 ,  0.46242517], dtype=float32)``\ ）
+
+-  If it's in the Hooper, N is 3, then the action value be chosen in \ ``[-1, 1]``\.
+
+Reward Space
+----------------
+
+-  Depending on the specific game content, the game score can vary very much and is usually a float value, which can be found in the performance section of the benchmark algorithm at the bottom.
+
+Others
+-----
+
+-  The end of the game is the end of the current environment episode
+
+KEY FACTS
+============
+
+1. Vector physical information input, empirically it is not advisable to subtract the mean value in doing norm
+
+2. Continuous action space
+
+3. Dense rewards
+
+4. Large variation in the scale of reward taking
+
+
+THE SPACE AFTER TRANSFORMATION（RL ENVIRONMENT）
+==============================================================
+
+
+
+Observation space
+-----------------------
+
+-  Basically no change
+
+
+Action space
+----------------
+
+-  Basically no transformation, still a continuous action space of size N, with a range of values\ ``[-1, 1]``\，size is \ ``(N, )``\ ，data type is \ ``np.float32``
+
+
+Reward space
+-------------
+
+-  Basically no change
+
+The above space can be represented using the gym environment space definition as follows
+
+.. code:: python
+
+   import gym
+
+
+   obs_space = gym.spaces.Box(low=-np.inf, high=np.inf, shape=(11, ), dtype=np.float64)
+   act_space = gym.spaces.Box(low=-1, high=1, shape=(3, ), dtype=np.float32)
+   rew_space = gym.spaces.Box(low=-np.inf, high=np.inf, shape=(1, ), dtype=np.float32)
+
+
+Others
+---------
+
+-  The \ ``info``\ returned by the environment \ ``step``\ method must contain the \ ``final_eval_reward``\ key-value pair, which represents the evaluation metrics for the entire episode, which in Mujoco is the cumulative sum of the rewards for the entire episode.
+
+
+OTHERS
+========
+
+Inert initialization
+----------------------
+
+To facilitate support for parallel operations such as environment vectorization, environment instances are generally implemented with inert initialization, i.e., the \``__init__`` method does not initialize the real original environment instance, but only sets the relevant parameters and configuration values, and initializes the concrete original environment instance when the \``reset`` method is called for the first time.
```

Comment:
1. inert initialization -> lazy initialization
2. all the format like __init__ is not shown properly